certification/pyxis/builder.go

```diff
@@ -67,68 +66,62 @@ func (b *certificationInputBuilder) Finalize() (*certificationInput, error) {
 	return &b.certificationInput, nil
 }
 
-// WithCertImageFromFile adds a pyxis.CertImage from a file on disk to the CertificationInput.
+// WithCertImage adds a pyxis.CertImage from the passed io.Reader to the CertificationInput.
 // Errors are logged, but will not halt execution.
-func (b *certificationInputBuilder) WithCertImageFromFile(filepath string) *certificationInputBuilder {
-	if err := b.storeCertImage(filepath); err != nil {
+func (b *certificationInputBuilder) WithCertImage(r io.Reader) *certificationInputBuilder {
+	if err := b.storeCertImage(r); err != nil {
 		log.Error(err)
 	}
 
 	return b
 }
 
-// WithPreflightResultsFromFile adds formatters.UserResponse from a file on disk to the CertificationInput.
+// WithPreflightResults adds formatters.UserResponse from the passed io.Reader to the CertificationInput.
 // Errors are logged, but will not halt execution.
-func (b *certificationInputBuilder) WithPreflightResultsFromFile(filepath string) *certificationInputBuilder {
-	if err := b.storePreflightResults(filepath); err != nil {
+func (b *certificationInputBuilder) WithPreflightResults(r io.Reader) *certificationInputBuilder {
+	if err := b.storePreflightResults(r); err != nil {
 		log.Error(err)
 	}
 
 	return b
 }
 
-// WithPreflightResultsFromFile adds the pyxis.RPMManifest from a file on disk to the CertificationInput.
+// WithRPMManifest adds the pyxis.RPMManifest from the passed io.Reader to the CertificationInput.
 // Errors are logged, but will not halt execution.
-func (b *certificationInputBuilder) WithRPMManifestFromFile(filepath string) *certificationInputBuilder {
-	if err := b.storeRPMManifest(filepath); err != nil {
+func (b *certificationInputBuilder) WithRPMManifest(r io.Reader) *certificationInputBuilder {
+	if err := b.storeRPMManifest(r); err != nil {
 		log.Error(err)
 	}
 
 	return b
 }
 
-// WithArtifactFromFile reads a file at path and binds it as an artifact to include
+// WithArtifactFromFile reads from the io.Reader and binds it as an artifact to include
```

Comment:
The method name in this comment needs to change to match what it is on line 102.